src/ZFS-SDK/Fsx.fs

```diff
@@ -26,7 +31,7 @@ open Infrastructure
 module Cost = Zen.Cost.Realized
 
 // Contract Arguments
-let contractId = ContractId (Version0, Hash.compute "%s"B)
+let contractId = ContractId (Version0, Hash.fromString "%s" |> Result.get)
```

Comment:
Would prefer that this case was handled with an exception rather than `Result.get`